login_ldap.php

```diff
@@ -11,16 +11,6 @@
   $content = "";
   $person = new People();
 
-  if ( isset($_GET['logout'])) {
```

Comment:
Moved logout code to new file, logout.php, so we're not duplicating this code and having to remember to maintain it in two different places.